packages/generator-rest/test/poc/generated/PriceCalculation/git_push.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
```

Comment:
Is there a way to not generate this and the other unused files?

Reply:
Yes, you have to add the file names into the `.openapi-generator-ignore` file.
The problem is you have to know all the unused files, but when updating versions, I see more files are generated.